### ISSUE TITLE
DataTable: Request caption support for accessibility purposes.

### ISSUE THREAD

**Issue by smckeetx** [primefaces/primefaces#2048]
## 1) Environment
- PrimeFaces version: 6.0
- Does it work on the newest PrimeFaces version? Unknown
- Application server + version: Tomcat 8.0.30
- Affected browsers: All

## 2) Expected behavior

Add &lt;caption&gt; to dataTable for accessibility purposes.

## 3) Actual behavior

Caption not supported

## 4) Steps to reproduce

N/A

## 5) Sample XHTML

N/A

## 6) Sample bean

N/A

**Comment by devilanne** [comment]
Yes! Please add caption support. We have to use extra scripting to add captions and it would make things so much easier if data tables supported a caption attribute. Captions are a must for accessibility.

**Comment by jxmai** [comment]
adding caption support is trivial but dealing with pagination is quite challenging from spec wise as we have no place to hold the upper pagination bar. leaving pagination will lead to the case when the table header renders under pagination.

**Comment by jxmai** [comment]
This was a failed attempt: https://github.com/primefaces/primefaces/pull/2855. Mostly there, but the pagination + header case had failed.

**Comment by melloware** [comment]
duplicate of https://github.com/primefaces/primefaces/issues/4725 so tracking it there.
